cmd/root/helpers/healthcheck_strategy_test.go

```diff
@@ -68,18 +64,16 @@ func TestTmpDirHC(t *testing.T) {
 // Tests the method returns the correct path
 func TestGetPathHC(t *testing.T) {
 	ddcfs := NewFakeFileSystem()
-	testStrat, err := NewHCCopyStrategy(ddcfs, &MockTimeService{Time: time.Now()})
-	if err != nil {
-		t.Errorf("error when creating copy strategy: %v", err)
-	}
+	tmpDir := t.TempDir()
+	testStrat := NewHCCopyStrategy(ddcfs, &MockTimeService{Time: time.Now()}, tmpDir)
 	// Test path for coordinators
-	expected := filepath.Join("tmp", "dir1", "random", testStrat.BaseDir, "log", "node1-C")
+	expected := filepath.Join(tmpDir, testStrat.BaseDir, "log", "node1-C")
```

Comment:
I was wondering about these tests - since we now use the hostname instead of the -C / -E suffix for ssh collections (we had to do this historically due to coordinator / executor identification in the HC tool)

We can probably remove these at a later date as a tech debt cleanup?

Reply:
well pretend those are the host names, there is no actual code that is tied to the old suffix names in the test...plus all of these would be removed at a later date because..there would be no health check strategy